### ISSUE TITLE
Fix number of fires in Toolik sample dataset

### ISSUE THREAD

**Issue by rarutter** [ua-snap/dvm-dos-tem#392]
There are (incorrectly) a lot of fires in the Toolik10x10 historic-explicit-fire.nc file.

**Comment by tobeycarman** [comment]
Indeed, that is because the file was generated with random data using the `create_region_input.py` script `fill_explicit_fire_file()` function. It is definitely not correct data!

**Comment by rarutter** [comment]
Yeah, we just realized it's limited by the fire mask - there's only one valid fire (which we added manually for testing).
